spec.html

```diff
@@ -903,27 +903,29 @@ <h1>Static Semantics</h1>
+        <li><em>Number</em>: IEEE 754-2019 double-precision floating point values.</li>
+        <li><em>BigInt</em>: Arbritary integer.</li>
       </ul>
 
-      <p>In the language of this specification, numerical values and operations (including addition, subtraction, negation, multiplication, division, and comparison) are distinguished among different numeric kinds using subscripts. The subscript <sub><dfn id="𝔽">𝔽</dfn></sub> refers to Numbers, and the subscript <sub><dfn id="ℝ">ℝ</dfn></sub> refers to mathematical values. A subscript is used following each numeric value and operation.</p>
-      <p>For brevity, the <sub>𝔽</sub> subscript can be omitted on Number values&mdash;a numeric value with no subscript is interpreted to be a Number. An operation with no subscript is interpreted to be a Number operation, unless one of the parameters has a particular subscript, in which case the operation adopts that subscript. For example, 1<sub>ℝ</sub> + 2<sub>ℝ</sub> = 3<sub>ℝ</sub> is a statement about mathematical values, and 1 + 2 = 3 is a statement about Numbers.</p>
-      <p>In general, when this specification refers to a numerical value, such as in the phrase, "the length of _y_" or "the integer represented by the four hexadecimal digits ...", without explicitly specifying a numeric kind, the phrase refers to a Number. Phrases which refer to a mathematical value are explicitly annotated as such; for example, "the mathematical value of the number of code points in ...".</p>
-      <p>It is not defined to mix Numbers and mathematical values in either arithmetic or comparison operations, and any such undefined operation would be an editorial error in this specification text.</p>
-      <p>The Number value 0, alternatively written 0<sub>𝔽</sub>, is defined as the double-precision floating point positive zero value. In certain contexts, it may also be written as *+0* for clarity.</p>
+      <p>In the language of this specification, numerical values and operations (including addition, subtraction, negation, multiplication, division, and comparison) are distinguished among different numeric kinds using subscripts. The subscript <sub><dfn id="𝔽">𝔽</dfn></sub> refers to Numbers, the subscript <sub><dfn id="ℝ">ℝ</dfn></sub> refers to mathematical values, and the subscript <sub><dfn id="ℤ">ℤ</dfn></sub> refers to BigInts. A subscript is used following each numeric value and operation.</p>
+      <p>For brevity, the <sub>ℝ</sub> subscript can be omitted on numeric values&mdash;a numeric value with no subscript is interpreted to be a mathematical value. An operation with no subscript is interpreted to be a mathematical value operation, unless one of the parameters has a particular subscript, in which case the operation adopts that subscript. For example, 1<sub>𝔽</sub> + 2<sub>𝔽</sub> = 3<sub>𝔽</sub> is a statement about Number values, and 1 + 2 = 3 is a statement about mathematical values.</p>
```

Comment:
I don't think this is a good way to say this, especially as we can use aliases as operands. I would say something much simpler, like

> Numeric operators such as +, &times;, =, and &ge; refer to those operations as determined by the type of the operands. When applied to mathematical values, the operators refer to the usual mathematical operations. When applied to Numbers, the operators refer to the relevant operations within IEEE 754-2019.

Do we use numeric operators on BigInts? If so, we may want to also say

> When applied to BigInts, the operators refer to the usual mathematical operations applied to the mathematical value of the BigInt.